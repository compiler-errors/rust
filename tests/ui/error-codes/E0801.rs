#![feature(return_type_notation)]

fn test()
where
    test(..): Send,
    //~^ ERROR expected associated function, found function `test`
    //~| ERROR return type notation not allowed in this position yet
{
}

fn main() {}
