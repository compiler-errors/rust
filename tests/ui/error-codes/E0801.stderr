error[E0801]: expected associated function, found function `test`
  --> $DIR/E0801.rs:5:5
   |
LL |     test(..): Send,
   |     ^^^^^^^^ not a associated function

error: return type notation not allowed in this position yet
  --> $DIR/E0801.rs:5:5
   |
LL |     test(..): Send,
   |     ^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0801`.
